src/pages/documentation/current-sdk-versions.md

```diff
@@ -100,7 +100,7 @@ Adobe Experience Platform Mobile SDK plugin for React Native supports React Nati
 
 Adobe Experience Platform Mobile SDK plugins for React Native are compatible with the [Android](#android) and [iOS](#ios) native libraries.
 
-<InlineAlert variant="info" slots="text"/>
+<InlineAlert variant="warning" slots="text"/>
```

Comment:
Why the warning block is black, not orange in the doc?

Reply:
I'm not quite sure what's going on there - we might want to bring this up to the Developer team.